pkg/testutils/endpoint.go

```diff
@@ -61,6 +73,7 @@ func (e *TestEndpoint) GetIdentity() identity.NumericIdentity   { return e.Ident
 func (e *TestEndpoint) GetSecurityIdentity() *identity.Identity { return e.Identity }
 func (e *TestEndpoint) GetNodeMAC() mac.MAC                     { return e.MAC }
 func (e *TestEndpoint) GetOptions() *option.IntOptions          { return e.Opts }
+func (e *TestEndpoint) IsHost() bool                            { return false }
```

Comment:
nit: So the endpoint returned by `NewTestHostEndpoint()` will return `false` to calls to `IsHost()`?